test/Microsoft.AspNetCore.Razor.Evolution.Test/Legacy/CSharpDirectivesTest.cs

```diff
@@ -751,6 +751,29 @@ public void OptionalDirectiveTokens_AreSkipped()
                     Factory.Span(SpanKind.Markup, " ", markup: false).Accepts(AcceptedCharacters.WhiteSpace)));
         }
 
+        [Theory]
```

Comment:
- no theories.
- Should ensure there's a test that validates that the whitespace after your string token doesn't get ignored.